src/init.c

```diff
@@ -409,6 +394,12 @@ shmem_internal_init(int tl_requested, int *tl_provided)
         goto cleanup;
     }
 
+    DEBUG_MSG("Local rank=%d, Num. local=%d, Shr. rank=%d, Num. shr=%d\n",
+              enable_local_ranks ? shmem_runtime_get_local_rank(shmem_internal_my_pe) : 0,
+              enable_local_ranks ? shmem_runtime_get_local_size() : 1,
+              shmem_internal_get_shr_rank(shmem_internal_my_pe),
+              shmem_internal_get_shr_size());
```

Comment:
Should we rename "local" rank/size to "node" rank/size?

Reply:
I do think I prefer that - "local" to me could mean something like NUMA-local, but we're specifically going for "compute node" locality, right?